Dockerfile

```diff
@@ -1,12 +1,50 @@
 FROM php:7.3-apache
```

Comment:
May be bump to 7.4 as well?

Reply:
My intention was to annotate the Dockerfile.  I would be fine taking this to the next level and properly configuring it for all the optional extensions but I feel I need a go-ahead before doing that.

Reply:
That would be great. Out of the scope for this PR however.